### PR TITLE
Integrate "Formik" into the project.

### PR DESCRIPTION
**Pull request by sergei-gilevich**
All `redux-form` forms are replaced with Formik forms in this Pull Request

### CODE REVIEW

File: packages/common/validation.js [hunk 1]
```diff
@@ -48,6 +47,9 @@ export const validateForm = (fValues, fSchema) => {
           validateForm1(values[v], schema[v], collector);
         }
       });
+    if (values.password && values.passwordConfirmation && values.password !== values.passwordConfirmation) {
```
Comment (evgenijolshanskij) [inline packages/common/validation.js]: Why is this checker here? What happened when comparable fields have different names?

File: packages/client/src/modules/post/containers/Post.spec.js [hunk 2]
```diff
@@ -338,7 +340,7 @@ describe('Posts and comments example UI works', () => {
     const editButtons = content.find('.edit-comment');
     expect(editButtons).has.lengthOf(6);
     editButtons.last().simulate('click');
-
+    //done();
```
Comment (vlasenko) [inline packages/client/src/modules/post/containers/Post.spec.js]: What is this?

File: packages/common/validation.js [hunk 3]
```diff
@@ -28,3 +32,28 @@ export const alphaNumeric = value =>
 // Phone number validation
 export const phoneNumber = value =>
   value && !/^(0|[1-9][0-9]{9})$/i.test(value) ? 'Invalid phone number, must be 10 digits' : undefined;
+
+export const validateForm = (fValues, fSchema) => {
```
Comment (vlasenko) [inline packages/common/validation.js]: why the naming `fValues`, `fSchema` here? should be `formValues`, `formSchema`

File: packages/common/validation.js [hunk 4]
```diff
@@ -28,3 +32,28 @@ export const alphaNumeric = value =>
...
+  let errors = {};
+  const validateForm1 = (values, schema, collector) => {
```
Comment (vlasenko) [inline packages/common/validation.js]: Names must be meaningful, why the name `validateForm1` here, it does not have meaning

**Comment by vlasenko** [comment]
The global question is why only part of kit examples was moved to `Formik`, why not all? Why it is not explained here, what are your intentions?

File: packages/common/validation.js (hunk 3 above)
Comment (evgenijolshanskij) [reply packages/common/validation.js]: Fixed

File: packages/common/validation.js (hunk 4 above)
Comment (evgenijolshanskij) [reply packages/common/validation.js]: That is an inner function and not visible from outside, that's why it was called in such way. Anyways, I have renamed it.

Comment (vlasenko) [reply packages/common/validation.js]: @evgenijolshanskij It doesn't matter, names must be meaningful at all times

**Comment by evgenijolshanskij** [comment]
The login form has been left untouched to provide an example of using 'redux-form'. The subscription form is going to be replaced shortly.

**Comment by vlasenko** [comment]
@evgenijolshanskij Lets not include examples of `redux-form`, since there is no anything specific in using `redux-form` with this starter kit

**Comment by evgenijolshanskij** [comment]
@vlasenko Ok, no problem.

**Pull request merged**